web/templates/stream_settings/confirm_stream_privacy_change_modal.hbs

```diff
@@ -0,0 +1,8 @@
+<div class="confirm-stream-privacy-modal">
+    <div class="new-style">
+        <p class="confirm-stream-privacy-info">
+            {{t "Changing the privacy of this steam will make the entire message history accessible according to the new
+              privacy configuration?"}}
```

Comment:
The original string in the issue ended with a `.`, not a `?`; is this change intentional? I think the `.` is a more natural choice.

Reply:
Thanks, I'll fix them right away.